src/libaktualizr/primary/eventsinterpreter.cc

```diff
@@ -29,11 +29,20 @@ std::shared_ptr<command::BaseCommand> EventsInterpreter::handle_cycle(event::Bas
   } else if (event.variant == "FetchMetaComplete") {
     return std::make_shared<command::CheckUpdates>();
   } else if (event.variant == "UpdateAvailable") {
-    return std::make_shared<command::StartDownload>(dynamic_cast<event::UpdateAvailable &>(event).updates);
+    auto upadte_available = dynamic_cast<event::UpdateAvailable &>(event);
```

Comment:
`upadte_available` instead of `update_available` in multiple places

Reply:
Thanks, fixed